transforms/CANFD.yaml

```diff
@@ -0,0 +1,150 @@
+transforms:
+  - !DeleteUselessEnums
+
+  #
+  - !DeleteEnumsWithVariants
+    variants:
+      0: CLR
+      1: SET
+
+  - !DeleteEnumsWithVariants
+    variants:
+      0: RESET
+      1: SET
+
+  # RESETASSERT is useless
+  - !DeleteEnumsWithVariants
+    variants:
+      0: NOP
+      1: ASSERT
+
+  # RESETSTKY is useless
+  - !DeleteEnumsWithVariants
+    variants:
+      0: NORES
+      1: RESET
+
+  # RESETSTKYCLR is useless
+  - !DeleteEnumsWithVariants
+    variants:
+      0: NOP
+      1: CLR
+
+  # TODO: Something better than a cursed rename as merge
+  - !MergeBlocks
+    from: CANFD0
+    main: CANFD0
+    to: GPRCM
```

Comment:
Wait you can do that? Well I may need to borrow this for some of the other peripherals...

Although is this the correct merge direction? This actually results in the GPRCM block existing and the fields from CANFD being merged into it.

I think the following is actually correct here?
```yaml
  - !MergeBlocks
    from: GPRCM
    main: CANFD0
    to: CANFD0
```